examples/camera_streamer/operators/oakd_camera/oakd_camera_op.py

```
@@ -421,226 +399,103 @@ def compute(self, op_input, op_output, context):
         try:
-            if self._output_format == OakdOutputFormat.RAW:
-                emitted = self._emit_raw_frames(op_output)
-            else:
-                emitted = self._emit_h264_packets(op_output)
+            emitted = False
+            for stream in self._streams.values():
+                if self._output_format == OakdOutputFormat.RAW:
+                    if self._emit_stream_raw(op_output, stream):
+                        emitted = True
+                else:
+                    if self._emit_stream_h264(op_output, stream):
```

Ruff (0.15.7)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)